src/api/schemas/user-schema.ts

```diff
@@ -0,0 +1,57 @@
+import { object, string } from 'zod';
+
+const createUserSchema = object({
+    body: object({
```

Comment:
I believe the user schema is missing the firebase_id because the error message does not mention it but it returns a db error that is due to the missing firebase_id:
![image](https://user-images.githubusercontent.com/64218770/195694525-062a08ab-6972-4ab6-9cc9-4ef74f7046a7.png)

Reply:
Ack, I failed to take into account that change from the last PR! Thank you!